test/e2e/storage/vsphere/vsphere_volume_cluster_ds.go

```diff
@@ -47,13 +47,19 @@ var _ = utils.SIGDescribe("Volume Provisioning On Clustered Datastore [Feature:v
 		namespace        string
 		scParameters     map[string]string
 		clusterDatastore string
+		nodeInfo         *NodeInfo
 	)
 
 	BeforeEach(func() {
 		framework.SkipUnlessProviderIs("vsphere")
 		Bootstrap(f)
 		client = f.ClientSet
 		namespace = f.Namespace.Name
+		nodeList := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
```

Comment:
same comment about re-getting nodeList here vs using it from TextContext.

Reply:
used
```
nodeInfo = GetReadySchedulableRandomNodeInfo()
```